Source/Atomic/UI/UIInput.cpp

```diff
@@ -427,7 +427,7 @@ void UI::HandleKey(bool keydown, int keycode, int scancode)
     }
     else
     {
-        InvokeKey(this, rootWidget_, 0, specialKey, mod, keydown);
+        InvokeKey(this, rootWidget_, keycode, specialKey, mod, keydown);
```

Comment:
Ok, one side effect of this change is that pressing TAB in an inspector field inserts a tab into the field instead of moving to the next widget.  There are likely other side effects. 

InvokeKey currently expects no keycode for when a special key is being invoked.  What about arrow keys is requiring the keycode to be passed?